integration_test.go

```diff
@@ -80,9 +82,64 @@ func argHandler(arg int32) (int32, error) {
 	}
 }
 
+func verifyBaggage(ctx context.Context) error {
+	span := opentracing.SpanFromContext(ctx)
+	if span == nil {
+		return errors.New("missing span")
+	}
+
+	spanCtx, ok := span.Context().(jaeger.SpanContext)
+	if !ok {
+		return errors.New("trace context is not from jaeger")
+	}
+	if !spanCtx.IsDebug() {
+		return errors.New("span context is not configured to submit traces")
+	}
+
+	val := span.BaggageItem("baggagekey")
+	if val == "" {
+		return errors.New("missing baggage")
+	} else if val != "baggagevalue" {
```

Comment:
nit: no need for `else` since the above case returns,
```go
val := span.BaggageItem(..)
if val == "" {
  return errors.New("missing baggage")
}
if val != "baggagevalue" {
  return errors.New("unexpected baggage")
}
return nil
```